<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>boost.git - boost</title>
<meta name='generator' content='cgit v0.10.2'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit-data/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://pkgs.fedoraproject.org/cgit/boost.git/atom/boost-1.48.0-python3.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/cgit/'><img src='/cgit-data/cgit.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/cgit/'>index</a> : <a title='boost.git' href='/cgit/boost.git/'>boost.git</a></td><td class='form'><form method='get' action=''>
<input type='hidden' name='id' value='bc9c5908b693593a53517ed7b3bf71df377c32c6'/><select name='h' onchange='this.form.submit();'>
<option value='boost-1_34_0-exp'>boost-1_34_0-exp</option>
<option value='f10'>f10</option>
<option value='f11'>f11</option>
<option value='f12'>f12</option>
<option value='f13'>f13</option>
<option value='f14'>f14</option>
<option value='f15'>f15</option>
<option value='f16'>f16</option>
<option value='f17'>f17</option>
<option value='f18'>f18</option>
<option value='f19'>f19</option>
<option value='f20'>f20</option>
<option value='f21'>f21</option>
<option value='f22'>f22</option>
<option value='f7'>f7</option>
<option value='f8'>f8</option>
<option value='f9'>f9</option>
<option value='master' selected='selected'>master</option>
<option value='pmachata/boost153'>pmachata/boost153</option>
<option value='pmachata/master-next'>pmachata/master-next</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>boost</td><td class='sub right'>Jesse Keating</td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/boost.git/'>summary</a><a href='/cgit/boost.git/refs/?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>refs</a><a href='/cgit/boost.git/log/boost-1.48.0-python3.patch'>log</a><a class='active' href='/cgit/boost.git/tree/boost-1.48.0-python3.patch?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>tree</a><a href='/cgit/boost.git/commit/boost-1.48.0-python3.patch?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>commit</a><a href='/cgit/boost.git/diff/boost-1.48.0-python3.patch?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>diff</a><a href='/cgit/boost.git/stats/boost-1.48.0-python3.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/boost.git/log/boost-1.48.0-python3.patch'>
<input type='hidden' name='id' value='bc9c5908b693593a53517ed7b3bf71df377c32c6'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/boost.git/tree/?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>root</a>/<a href='/cgit/boost.git/tree/boost-1.48.0-python3.patch?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>boost-1.48.0-python3.patch</a></div><div class='content'>blob: 249906aa10056096f218a71909d425e9e415a7bd (<a href='/cgit/boost.git/plain/boost-1.48.0-python3.patch?id=bc9c5908b693593a53517ed7b3bf71df377c32c6'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
</pre></td>
<td class='lines'><pre><code>diff -up boost_1_48_0/libs/python/src/CMakeLists.txt\~ boost_1_48_0/libs/python/src/CMakeLists.txt
--- boost_1_48_0/libs/python/src/CMakeLists.txt~	2012-05-02 20:29:17.095570094 +0200
+++ boost_1_48_0/libs/python/src/CMakeLists.txt	2012-05-02 20:29:27.645898988 +0200
@@ -29,7 +29,8 @@ endif (PYTHON_DEBUG_LIBRARIES AND BUILD_
 # Always build the non-debug variants of the boost_python library
 set(BUILD_PYTHON_NODEBUG ON)
 
-boost_add_library(python
+colormsg (GREEN "+-- BOOST_PYTHON_SUFFIX=${BOOST_PYTHON_SUFFIX}")
+boost_add_library(python${BOOST_PYTHON_SUFFIX}
   numeric.cpp 
   list.cpp 
   long.cpp 

Diff finished.  Wed May  2 20:29:38 2012
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.10.2 at 2015-04-11 18:09:01 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
